src/components/pod/pod.pw.tsx

```diff
@@ -0,0 +1,619 @@
+
+        const elem = podBlock(page);
+        await checkCSSOutline(elem, px, "border", style, color);
+      });
+    }
+  );
+
+  specialCharacters.forEach((children) => {
+    test(`should render with ${children} as children`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PodExample>{children}</PodExample>);
+
+      page.getByText(children);
+      await expect(podContent(page)).toHaveCSS("text-align", "left");
+    });
+  });
+
+  specialCharacters.forEach((className) => {
+    test(`should render with ${className} as className`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<PodExample className={className} />);
+
+      await containsClass(podComponent(page), className);
+    });
+  });
+
+  ([
+    [SIZE.EXTRASMALL, 120, 66],
+    [SIZE.SMALL, 120, 66],
+    [SIZE.MEDIUM, 136, 82],
+    [SIZE.LARGE, 184, 98],
+    [SIZE.EXTRALARGE, 216, 130],
+  ] as [PodProps["size"], number, number][]).forEach(
+    ([size, expectedHeight, expectedWidth]) => {
+      test(`should render with ${size} as size and ${expectedHeight} as height and ${expectedWidth} as width`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<PodExample size={size} />);
+
+        await assertCssValueIsApproximately(
+          podBlock(page),
+          "height",
+          expectedHeight
+        );
+
+        await assertCssValueIsApproximately(
+          podBlock(page),
+          "width",
+          expectedWidth
+        );
+      });
+    }
+  );
+
+  ([
+    ["primary", "rgb(255, 255, 255)", "none"],
+    ["secondary", "rgb(242, 245, 246)", "none"],
+    ["tertiary", "rgb(237, 241, 242)", "none"],
+    ["tile", "rgb(255, 255, 255)", "rgba(2, 18, 36, 0.2) 0px 2px 3px 0px"],
+    ["transparent", "rgba(0, 0, 0, 0)", "none"],
+  ] as [PodProps["variant"], string, string][]).forEach(
+    ([variant, color, boxShadow]) => {
+      test(`should render with ${variant} as variant when color is ${color} and boxShadow is ${boxShadow}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<PodExample variant={variant} />);
+
+        await expect(podBlock(page)).toHaveCSS("background-color", `${color}`);
+
+        await expect(podBlock(page)).toHaveCSS("box-shadow", boxShadow);
+      });
+    }
+  );
+
+  specialCharacters.forEach((title) => {
+    test(`should render with ${title} as title`, async ({ mount, page }) => {
+      await mount(<PodExample title={title} />);
+
+      await expect(podTitle(page)).toHaveText(title);
+    });
+  });
+
+  variantTypes1.forEach((variantType) => {
+    test(`should render title node when node is passed as ${variantType} variant`, async ({
+      mount,
+      page,
+    }) => {
+      const title = "title";
+      await mount(<PodWithVariantTypes title={title} />);
+
+      const variantLocator = page.locator('[data-element="title"]');
+
+      await expect(variantLocator).toContainText(title);
+    });
+  });
```

Comment:
I'm not sure if variant has been tested here properly. All of the tests looks similar to me so could you please have a look.